src/features/unified-share-modal/__tests__/UnifiedShareForm.test.js

```diff
@@ -73,6 +73,12 @@ describe('features/unified-share-modal/UnifiedShareForm', () => {
                 {...props}
             />,
         );
+    const mockUpsellInlineNotice = (
+        <div className="upsellInlineNotice" data-testid="upsellinlinenotice">
```

Comment:
Can we remove the `className` here since the wrapping div within USM will wrap with that `className`.  Probably dont need the `data-testid` anymore if you select by className